src/main/java/utility/DateAndTime.java

```diff
@@ -0,0 +1,14 @@
+package utility;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateAndTime {
+	public static String getDateAndTime() {
+		LocalDateTime time=LocalDateTime.now();
+		DateTimeFormatter type= DateTimeFormatter.ofPattern("dd-MM-yyyy-HH-mm-ss");
+		String datetime=time.format(type);
```

Comment:
datetime variable name should be like dateTime

Reply:
Done.